build.ps1

```diff
@@ -46,6 +46,8 @@ function bazel() {
 # which is a workaround for this problem.
 bazel shutdown
 
+bazel clean --expunge
```

Comment:
should that say
```
`-`-expunge
```
?

Reply:
pretty sure this worked the last couple of times, don’t ask me why

Reply:
So... Not asking why it works without the backticks, as requested, but do you have any idea why the `bazel build` commands on lines 53 & 57 need them, then?

Reply:
tbh, no. I don’t remember the error we ran into without those escapes but at least for expunge they don’t seem to be necessary.